### ISSUE TITLE
without internet

### ISSUE THREAD

**Issue by axroromonov** [Lyokone/flutterlocation#920]
Is it possible to get exact location through GPS even without internet through this package?

**Comment by Ockersz** [comment]
Did anyone find a fix for this? If Network is not available the Location cannot be fetched and doesn't even time out.

**Comment by Lyokone** [comment]
Yes, GPS provides a position without internet. Only the network based (coarse) provider needs connectivity. Same answer as #833.

Closing as a usage question.
